index.html

```diff
@@ -209,6 +209,35 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
         </ul>
       </div>
     </section>
+    <section class="last-page">
+      <div class="contact-me">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-info">If you have an application you are interested in developing, a feature that
+          you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="form" action="https://formspree.io/f/mnqlvbjg" method="POST">
+          <ul>
+            <li class="name">
+              <label for="name"></label>
+              <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+            </li>
+
+            <li class="email">
+              <label for="email"></label>
+              <input type="email" id="email" name="user_email" placeholder="Email address" required>
+            </li>
+
+            <li class="textarea">
+              <label for="msg"></label>
+              <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500"
+                required></textarea>
+            </li>
+
+          </ul>
+          <button class="button" type="submit">Get in touch</button>
```

Comment:
- [ ] When I press the button, this doesn't change the color as the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) requires. Please, consider changing the color on the button when this is pressed on the form. 🐶 
![image](https://user-images.githubusercontent.com/20563252/125369821-b04b3e80-e342-11eb-9808-cbef30de0478.png)

Reply:
I made it enabled as Figma design should I use hover in this case. I think I have used the proper attribute.

Reply:
- Kindly follow the requirements of your last reviewer. Isn't working that effect running locally or in your live version.
- Add .vscode to your .gitignore file.